docs/source/development/index.rst

```diff
@@ -91,7 +99,28 @@ on the main master branch to ease merging:
 
 .. code:: bash
 
-    $ git remote add upstream https://github.com/mwclient/mwclient.git
-    $ git rebase upstream master
+    $ git rebase master
+
+When it is ready, push your branch to your remote:
+
+.. code:: bash
+
+    $ git push -u fork my-branch
+
+Then you can open a pull request on GitHub. You should see a URL to do this
+when you push your branch.
```

Comment:
Just to be sure, was it intentional that you did not break the second sentence into its own line?

Reply:
Oh, right — sorry, I should've been more explicit in my [first comment](https://github.com/mwclient/mwclient/pull/331#discussion_r1590804140). I am not adamant about semantic line breaks, but I like to encourage them when possible.